src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2445,6 +2445,44 @@ def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certifi
                                                 certificate_envelope=kv_cert_def)
 
 
+def sync_kv_cert(cmd, resource_group_name, name, certificate_name, key_vault):
+    Certificate = cmd.get_models('Certificate')
+    client = web_client_factory(cmd.cli_ctx)
+    webapp = client.web_apps.get(resource_group_name, name)
+    if not webapp:
+        err_msg = "'{}' app doesn't exist in resource group {}".format(name, resource_group_name)
+        raise ResourceNotFoundError(err_msg)
```

Comment:
Why do we need this specific check? Is this something we do on other commands? We should have a common validator that we use on other commands already that should check this.

Reply:
This was already done in the import certificate command, but I can remove it from both.